src/test/compile-fail/invalid-inline.rs

```diff
@@ -21,4 +21,8 @@ fn b() {
 fn c() {
```

Comment:
Shouldn't we check the `inline` attribute for functions that aren't translated? I don't want people to create functions with an `inline` attribute that explode when a child crate finally instantiates them.